### ISSUE TITLE
Difficulties with 5.4.6 release

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#924]
If we are to release 5.4.6 this summer then we need to be able to build the package.  I just tried to switch to the 5.4 branch and when running make gmt_release I get this:

```
pwessel@macnut:~/GMTdev/gmt-dev/build-> cmake --build . --target gmt_release
ninja: error: '../doc/rst/svn_export_release', needed by 'doc/rst/CMakeFiles/_html_release', missing and no known rule to make it

```
Anyone know what changed or what needs to be added?  We discussed fixing the build so that it would include the PDF (unlike 6 where there are no PDF documentation).

**Comment by PaulWessel** [comment]
I wonder if instead of messing with the old CMake files in 5.4 it might be simpler to back port the current CMake files from master and use those to build the 5.4 docs? What say you, @leouieda?

**Comment by leouieda** [comment]
@PaulWessel that might be simpler. The problem would be trying to actually backport the commit that changed them instead of manually copying over the files. There is a way in git to apply a commit from master into another branch. It's called `git cherrypick`. But we'd need to know the exact commit that has our changes.

**Comment by seisman** [comment]
Closed by #1022.
